Makefile

```diff
@@ -0,0 +1,33 @@
+export GO15VENDOREXPERIMENT:=1
+export CGO_ENABLED:=0
+export GOARCH:=amd64
+
+SHELL:=$(shell which bash)
+LOCAL_OS:=$(shell uname | tr A-Z a-z)
+GOFILES:=$(shell find . -name '*.go' | grep -v -E '(./vendor)')
+GOFLAGS=
+
+all:
+	#_output/bin/$(LOCAL_OS)/binary
+	#_output/bin/linux/binary
+
+check:
+	@gofmt -l -s $(GOFILES) | read; if [ $$? == 0 ]; then gofmt -s -d $(GOFILES); exit 1; fi
+	@go vet $(shell go list ./... | grep -v '/vendor/')
```

Comment:
nit: `go list ./...` doesn't match vendor anymore https://golang.org/doc/go1.9#vendor-dotdotdot

Reply:
fixed this